files/grest/rpc/pool/pool_calidus_keys.sql

```diff
@@ -51,7 +51,7 @@ AS $$
   ) AS x
   WHERE is_valid=true
     AND length(x.calidus_key)=64
-  ORDER BY x.pool_id_bech32, x.calidus_nonce DESC;
+  ORDER BY x.pool_id_bech32, x.calidus_nonce DESC, x.block_time ASC;
```

Comment:
should be tx.id instead of block_time - as there can be multiple tx for same pool within same block?